content/40.units_and_quantities.md

```diff
@@ -1,269 +1,682 @@
+most cases, see below for exceptions):
+
+```python
+>>> print(momentum_y.to("degC"))
+Traceback (most recent call last):
+  File "<stdin>", line 1, in <module>
+  File "/Users/jzuhone/mambaforge/envs/py311/lib/python3.11/site-packages/unyt/array.py", line 947, in to
+    return self.in_units(units, equivalence=equivalence, **kwargs)
+           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  File "/Users/jzuhone/mambaforge/envs/py311/lib/python3.11/site-packages/unyt/array.py", line 874, in in_units
+    (conversion_factor, offset) = self.units.get_conversion_factor(
+                                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  File "/Users/jzuhone/mambaforge/envs/py311/lib/python3.11/site-packages/unyt/unit_object.py", line 694, in get_conversion_factor
+    return _get_conversion_factor(self, other_units, dtype)
+           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  File "/Users/jzuhone/mambaforge/envs/py311/lib/python3.11/site-packages/unyt/unit_object.py", line 939, in _get_conversion_factor
+    raise UnitConversionError(
+unyt.exceptions.UnitConversionError: Cannot convert between 'Msun*kpc/Myr' (dim '(length)*(mass)/(time)') and '°C' (dim '(temperature)').
+```
+
+#### Equivalencies {#sec:equivalencies}
+
+In certain circumstances, conversion from one quantity to another with different dimensions is
+desired. This is the case for a number of frequently encountered, physically-motivated
+transformations which involve physical constants, some examples of which are:
+
+* Conversions between temperature and energy via $E = k_BT$
+* Conversions between wavelength, frequency, and energy for light via $E = h\nu = hc/\lambda$
+* Conversions between mass and energy via $E = mc^2$
+* Conversions between density and number density via $n = \rho/({\mu}m_p)$
+
+As described above, conversions between quantities in different units is generally not allowed, but
+these special cases can be handled using unit equivalencies in `yt`. This is done using the
+`to_equivalent` method, which takes the unit in a different dimension that you want to convert to,
+and the equivalence you would like to use: 
+
+```python
+>>> import unyt as u
+>>> kT = 5.0*u.keV
+>>> # converting kT (energy) to T (temperature)
+>>> print(kT.to_equivalent("K", "thermal"))
+34813557.843240075 K
+>>> # converting kT (energy) to Angstrom (wavelength)
+>>> print(kT.to_equivalent("angstrom", "spectral"))
+4.132806438406553 Å
+>>> # print the proton's compton wavelength in picometers 
+>>> print(u.mp.to_equivalent("pm", "compton"))
+0.00132141 pm
+```
+
+These equivalencies are straightforward--in the case of others, more information may be required.
+For example, to convert temperature to sound speed via $c_s = \sqrt{\gamma{k_BT}/(\mu{m_p})}$, there
+are two parameters which may be optionally changed, $\gamma$ and $\mu$, which are the ratio of
+specific heats and the mean molecular weight, respectively:
```

Comment:
```suggestion
specific heat and the mean molecular weight, respectively:
```

Reply:
In this context, it is in fact "heats" (at least in English) because it's a ratio of C_p to C_v and each is a specific "heat"

Reply:
oh, I must have misread it